CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 <!-- ## [Unreleased] -->
 
-- Raise `peerDependency` on `stylelint` to 9.4.0 to accomodate the `linebreaks` rule.
+- Raise `peerDependency` on `stylelint` to 9.4.0 to accomodate the `linebreaks` rule. ([#46](https://github.com/Shopify/stylelint-config-shopify/pull/46))
```

Comment:
👍

Reply:
Should it be bumped to 9.5.0, as that's what's being used in stylelint-prettier v1.0.6?